src/Accounts/Accounts.Test/ErrorResolutionTests.cs

```diff
@@ -29,6 +31,56 @@ namespace Microsoft.Azure.Commands.Profile.Test
 {
     public class ErrorResolutionTests
     {
+        [Cmdlet("Mock", "Resolve" + ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Error", DefaultParameterSetName = ResolveError.AnyErrorParameterSet)]
+        class MockResolveError : ResolveError
+        {
+            new public void WriteInformation(object messageData, string[] tags)
+            {
+                ICommandRuntime2 commandRuntime2 = CommandRuntime as ICommandRuntime2;
+                InformationRecord informationRecord = new InformationRecord(messageData, String.Empty);
+
+                if (tags != null)
+                {
+                    informationRecord.Tags.AddRange(tags);
+                }
+                commandRuntime2.WriteInformation(informationRecord);
+            }
+
+            public override void ExecuteCmdlet()
```

Comment:
I'd like to the reason why ExecuteCmdlet() method needs to be overridden.

Reply:
> I'd like to the reason why ExecuteCmdlet() method needs to be overridden.

Change the implementation by using a virtual method wrapping the WriteInformation.